src/ProjectOrigin.WalletSystem.Server/ViewModels/CertificateViewModel.cs

```diff
@@ -1,20 +1,22 @@
 using Google.Protobuf.WellKnownTypes;
+using ProjectOrigin.WalletSystem.Server.Models;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 
-namespace ProjectOrigin.WalletSystem.Server.Models;
+namespace ProjectOrigin.WalletSystem.Server.ViewModels;
 
 public record CertificateViewModel
 {
-    public required Guid Id { get; init; }
+    public required Guid CertificateId { get; init; }
     public required string RegistryName { get; init; }
     public required GranularCertificateType CertificateType { get; init; }
     public required string GridArea { get; init; }
     public required DateTimeOffset StartDate { get; init; }
     public required DateTimeOffset EndDate { get; init; }
+    public required uint Quantity { get; init; }
     public List<CertificateAttribute> Attributes { get; } = new();
-    public List<SliceViewModel> Slices { get; } = new();
```

Comment:
Slices are now removed from certificates?

Reply:
yes, the slices was in this regards only used to calculate the sum of the certificate, they are now summed in the database instead.